CS/CASESTUDY1/Vehicle.java

```diff
@@ -0,0 +1,82 @@
+
+public class Vehicle {
```

Comment:
ACTION: Vechile class will be abstract class contain abstract methods of common features of bike and car. When you will extend it into Car and Bike class than you will define all the abstract methods and additional features according to the vehicle. Getter Setter is not to be used here.